bin/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+if [ -f "${DOCKER_CERT_PATH}/ca.pem" ]
```

Comment:
For now we'd typically set those in the setup script (as annoying as that is). It's a v3 feature to allow scripts to update the environment though.